roles/custom/matrix-synapse/defaults/main.yml

```diff
@@ -395,9 +395,16 @@ matrix_synapse_event_cache_size: "100K"
 # Controls cache sizes for Synapse.
 # Raise this to increase cache sizes or lower it to potentially lower memory use.
 # To learn more, see:
+# - https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html#caching
 # - https://github.com/matrix-org/synapse#help-synapse-eats-all-my-ram
 # - https://github.com/matrix-org/synapse/issues/3939
-matrix_synapse_caches_global_factor: 0.5
+matrix_synapse_caches_global_factor: 10
+matrix_synapse_caches_expire_caches: true
+matrix_synapse_caches_cache_entry_ttl: 30m
+matrix_synapse_caches_sync_response_cache_duration: 2m
+matrix_synapse_caches_autotuning_max_cache_memory_usage: 2048M
+matrix_synapse_caches_autotuning_target_cache_memory_usage: 1024M
+matrix_synapse_caches_autotuning_min_cache_ttl: 5m
```

Comment:
Maybe you want to make autotuning optional, like make a `matrix_synapse_caches_autotuning_enabled` and set it to `false` by default.

Then you wouldn't need to find values that work for everyone in every possible setup.  If a user is going to bother to enable autotune, then you can assume they will also put in the work to choose reasonable values.

Reply:
That is a resonable proposal to solve that concern. But we would still need to solve that this branch doesnt correctly deploy for unknown reasons. Like try to deploy this branch to a test system and it should complain.